algorithms/merge-sort.js

```diff
@@ -1 +1,33 @@
+function mergeSort(arr) {
+  let lHalf = arr.slice(0, arr.length / 2);
+  let rHalf = arr.slice(arr.length / 2, arr.length);
+  sort(lHalf);
+  sort(rHalf);
```

Comment:
`mergeSort` is supposed to be recursive and break down the array until it's a single element array. then as you combine the arrays, you add them sorted. so the two `sort` calls should `mergeSort` calls. Then you will have to change `merge` to combine the two arrays back in order.